ynr/apps/moderation_queue/review_required_helper.py

```diff
@@ -216,6 +216,26 @@ def needs_review(self):
             return self.Status.UNDECIDED
 
 
+class RevertedEditByAnotherUser(BaseReviewRequiredDecider):
+    def review_description_text(self):
+        return "Edit reverted by another user"
+
+    def needs_review(self):
+        from candidates.models import LoggedAction
+
+        if self.logged_action.action_type == "person-revert":
+            update_qs = LoggedAction.objects.filter(
+                person=self.logged_action.person, action_type="person-update"
+            ).order_by("updated")
+            # filter out edits that have the current user as the user
+            update_qs = update_qs.exclude(user=self.logged_action.user)
+            # find the last edit made by another user
+            if update_qs.exists():
+                if self.logged_action.user != update_qs.last().user:
```

Comment:
Re: [previous comment](https://github.com/DemocracyClub/yournextrepresentative/pull/1670#discussion_r773177558) I think the problem with this is that as long as there has been previously been an update by any other user, then a revert will always be flagged, regardless of the user making the revert. E.g with the current conditionals:
- User A makes a change
- User B makes a change
- User B reverts their change - this will get flagged for review
Or:
- User A makes a change
- User B makes a change
- User A reverts their change - this will get flagged for review

I'm not really sure how to get around this though other than getting in to comparing the exact change that is being reverted. But I dont think that would be possible without storing further information on the LoggedAction when it is created.

An easier alternative might be to only flag it for review when it is a revert of the **previous** update? So that would stop someone making a change and then another user attempting to revert it right after. I think this would be done:
```
if self.logged_action.action_type == "person-revert":
    last_update = self.logged_action.person.loggedaction_set.filter(
        action_type=ActionType.PERSON_UPDATE,
    ).order_by("updated").last()

    if last_update.user != self.logged_action.user:
        return self.Status.NEEDS_REVIEW
return self.Status.UNDECIDED
```
Although I'm not really sure if this is actually helpful ultimately!

Reply:
If we want to detect edit wars (or revert wars), then I suggest:
```
If the edit type is a revert:
  Look for edits in the last 24 hours that are also reverts:
     if there are more than [NUMBER] (2? 3?):
        flag this edit

```

It might not matter who the users are...we want to know if there are a lot of reverts going on

Reply:
@symroe to your suggestion, do you mean 'Look for edits in the last 24 hours that are also revert' on the same candidate?